learn/cloud/analytics.mdx

```diff
@@ -0,0 +1,145 @@
+
+Meilisearch Cloud will begin processing your request. The "Analytics" section will update when the feature is enabled.
+
+### Update URLs in your application
+
+When you enable analytics, Meilisearch Cloud changes your project's API URL. Meilisearch Cloud is only able to track metrics for queries sent to this URL. 
+
+Update your application so all API requests point to the new URL:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com/indexes/products/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "green socks" }'
+```
+
+**The previous API URL will remain functional**, but requests targeting it will not send any data to the analytics interface.
+
+### Configuring click-through rate and average click position
+
+To track metrics like click-through rate and average click position, Meilisearch Cloud needs to know when users click on search results.
+
+Every time a user clicks on a search result, your application must send a `click` event to the `POST` endpoint of Meilisearch Cloud analytics route:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer DEFAULT_SEARCH_API_KEY'
+  --data-binary '{ 
+    "eventType": "click",
+    "eventName": "Search result clicked",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+#### The `click` event object
+
+The `click` event must deliver an object with the following fields:
+
+```json
+{
+  "eventType": "click",
+  "eventName": "Search Result Clicked",
+  "indexUid": "products",
+  "objectID": "0",
+  "position": 0
+}
+```
+
+- `eventType`: a string indicating this is a `click` event
+- `eventName`: a string describing the event
+- `indexUid`: a string indicating the clicked document's index
+- `objectID`: a string indicating the clicked document's primary key
+- `position`: an integer indicating the clicked document's position in the list of search results
+
+### Configuring conversion rate
+
+To track conversion rate, first identify what should count as a conversion for your application. For example, in a web shop, a conversion might be a user finalizing the checkout process. 
+
+Once you have established what counts as a conversion in your application, configure it to send a `conversion` event to the `POST` endpoint of Meilisearch Cloud analytics route:
+
+```sh
+curl \
+  -X POST 'http://edge.meilisearch.com' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer Meilisearch_API_Key'
+  --data-binary '{ 
+    "eventType": "conversion",
+    "eventName": "Product Added to Cart",
+    "indexUid": "products",
+    "objectID": "0",
+    "position": 0
+  }'
+```
+
+It is not possible to associate multiple `conversion` events with the same search.
```

Comment:
that's weird. the way I see it, a given search could have "added to card" and then later "purchased". Or is it a misunderstanding on my end? 🤔

that being said if Bruno / Quentin agreed on the technical correctness of this, it's outside of the scope of this PR

Reply:
Ok, I verified with Quentin and here's what's happening

- you can send multiple conversion events (with different event names), and the database will store them
- the current UI of the Cloud, indeed, is not able to display the per event stats, so it would be confusing

so from my understanding the info is not exactly "correct", but it's probably the right advise to give our users